### PR TITLE
modify a question in FAQ.

### PR DESCRIPTION
**Pull request by dbaoutdo**

### CODE REVIEW

**Comment by dbaoutdo** [comment]
@sunhao2017  PTAL

**Comment by ngaut** [comment]
@lilin90

File: FAQ.md [hunk 1]
```diff
@@ -522,7 +522,7 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
 
 TiDB 目前暂时不支持 `select into outfile`，可以通过以下方式导出 TiDB 数据：参考 [MySQL使用mysqldump导出某个表的部分数据](http://blog.csdn.net/xin_yu_xin/article/details/7574662)，使用 mysqldump 加 where 条件导出，使用 MySQL client 将 select 的结果输出到一个文件。
 
-### 4.1.6 DB2、Oracle 这些数据库有没办法通过 OGG 同步到 TiDB。
+### 4.1.6 DB2、Oracle 这些数据库有那些方式可以同步到 TiDB？
```
Comment (lilin90) [inline FAQ.md]: 1. 那些 -> 哪些
2. Suggest a more formal expression:
-> 4.1.6 DB2、Oracle 这些数据库可以通过哪些方式同步到 TiDB？

File: FAQ.md [hunk 2]
```diff
@@ -522,7 +522,7 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
...
 
 DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法是程序实现、Oracle 官方工具 OGG、Gateway（透明网关）、CDC（Change Data Capture），导出（Spool）成文本然后进行 Load，第三方工具，目前看 OGG 最为合适。
```
Comment (queenypingcap) [inline FAQ.md]: I think we can create a document for this question.

Comment (queenypingcap) [inline FAQ.md]: What does 程序实现 mean？What 程序？

**Comment by dbaoutdo** [comment]
@lilin90  pls review

File: FAQ.md [hunk 3]
```diff
@@ -522,9 +522,14 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
...
-
-DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法是程序实现、Oracle 官方工具 OGG、Gateway（透明网关）、CDC（Change Data Capture），导出（Spool）成文本然后进行 Load，第三方工具，目前看 OGG 最为合适。
+### 4.1.6 DB2、Oracle 数据库如何迁移到 TiDB？
+
+DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法有：
+- 使用 Oracle 官方迁移工具，如 OGG、Gateway（透明网关）、CDC（Change Data Capture）。
+- 自研数据导出导入程序实现。
+- 导出（Spool）成文本文件，然后通过 Load infile 进行导入。
+- 第三方数据迁移工具。
+目前看来 OGG 最为合适。
```
Comment (lilin90) [inline FAQ.md]: Leave a blank line between line 531 & line 532, otherwise, the contents are displayed together.

File: FAQ.md [hunk 4]
```diff
@@ -522,9 +522,15 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
...
 
-### 4.1.6 DB2、Oracle 这些数据库有没办法通过 OGG 同步到 TiDB。
+### 4.1.6 DB2、Oracle 数据库如何迁移到 TiDB？
 
-DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法是程序实现、Oracle 官方工具 OGG、Gateway（透明网关）、CDC（Change Data Capture），导出（Spool）成文本然后进行 Load，第三方工具，目前看 OGG 最为合适。
+DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法有：
+- 使用 Oracle 官方迁移工具，如 OGG、Gateway（透明网关）、CDC（Change Data Capture）。
+- 自研数据导出导入程序实现。
+- 导出（Spool）成文本文件，然后通过 Load infile 进行导入。
+- 第三方数据迁移工具。
```
Comment (lilin90) [inline FAQ.md]: Use verbs at the beginning and make them consistent:
-> 使用第三方数据迁移工具。

**Comment by lilin90** [comment]
LGTM

**Pull request merged**